test/mocks/mockPlugin.js

```diff
@@ -22,15 +23,28 @@ module.exports = class MockPlugin extends EventEmitter2 {
     return 'test.example.alice'
   }
 
-  sendRequest () {
+  sendRequest (request) {
+    debug('send request %j', request)
     return Promise.resolve(null)
   }
 
-  sendTransfer () {
+  sendTransfer (transfer) {
+    debug('send transfer %j', transfer)
     return Promise.resolve(null)
   }
 
-  rejectIncomingTransfer () {
-    return Promise.resolve(null)
+  registerTransferHandler (handler) {
+    if (this._handler) {
+      throw new Error('Mock plugin already has a transfer handler')
+    }
+    this._handler = handler
+  }
+
+  deregisterTransferHandler (handler) {
+    this._handler = null
   }
 }
+
+MockPlugin.version = 2
```

Comment:
Should this be `lpiVersion` (see https://github.com/interledgerjs/ilp-compat-plugin/blob/d8331126a92ad2b65a7e7c4513ce44c13d485340/index.js#L31)?